perfkitbenchmarker/providers/azure/azure_network.py

```diff
@@ -457,7 +457,8 @@ def __init__(self, spec):
     self.location = util.GetLocationFromZone(self.zone)
     # With dedicated hosting and/or an availability zone, an availability set
     # cannot be created
-    if FLAGS.dedicated_hosts or util.GetAvailabilityZoneFromZone(self.zone):
+    if FLAGS.dedicated_hosts or util.GetAvailabilityZoneFromZone(self.zone) or \
```

Comment:
Back slash is not glint compliant. Recommend putting conditions in parenthesis instead.